assets/strings/spa.json

```diff
@@ -64,5 +49,19 @@
+    "Install_Apertium_Para": "¿Estás experimentando respuestas lentas? Nuestros servidores pueden estar sobrecargados. Aprenda cómo <a href='http://wiki.apertium.org/wiki/Installation' target='_blank' rel='noopener'> instalar Apertium localmente </a>.",
+    "Translate_Webpage": "Traducir una página web",
+    "What_Is_Apertium": "<p> Apertium es una <b> plataforma de traducción automática gratuita / de código abierto </b>, inicialmente destinada a pares de idiomas relacionados, pero se amplió para tratar con pares de idiomas más divergentes (como inglés-catalán). La plataforma proporciona </p> <ol> <li> un motor de traducción automática independiente del idioma </li> <li> herramientas para administrar los datos lingüísticos necesarios para construir un sistema de traducción automática para un par de idiomas dado y </li> <li> datos lingüísticos para un número creciente de pares de idiomas. </li> </ol> <p> Apertium le da la bienvenida a los nuevos desarrolladores: si cree que puede mejorar el motor o las herramientas, o desarrollar datos lingüísticos para nosotros, no lo haga. dude en <a data-toggle='modal' data-target='#contactModal' data-text='Help_Improve' data-keyboard='true' data-dismiss='modal' style='cursor: pointer'> contáctenos </a>. </p>",
+    "Translation_Help": "Ingrese texto o URL para traducir.",
+    "Multi_Step_Translation": "<a href='https://en.wikipedia.org/wiki/Pivot_language' target='_blank' rel='noopener'> Traducción de varios pasos </a> (experimental)",
+    "Drop_Document": "Soltar un documento",
+    "Documentation_Para": "Se puede encontrar documentación en nuestro <a href='http://wiki.apertium.org/'> Wiki </a> bajo el objetivo <a href = 'http://wiki.apertium.org/wiki/Documentation' = '_ blank' rel = 'noopener'> Subpágina de documentación </a>. Hemos publicado varios documentos de conferencias y artículos de revistas, una lista de los cuales se puede encontrar en la Wiki en <a href = 'http://wiki.apertium.org/wiki/Publications' target = '_ blank' rel = 'noopener' > Publicaciones </a>.",
+    "Install_Apertium": "Instalar Apertium",
+    "Supported_Formats": "Formatos admitidos: .txt, .rtf, .odp, .ods, .odt (LibreOffice / OpenOffice), .xlsx, .pptx, .docx (Microsoft Office 2003 y más reciente). <br/> <i> Los archivos .doc y pdf de Word 97 no son compatibles. </i>"
```

Comment:
Who translated these?

Reply:
I did, with the help of a translator. I found these missing from the Spanish translation.